SoftLayer/CLI/file/snapshot/enable.py

```diff
@@ -0,0 +1,58 @@
+# snapshot_enable.py
+"""Create a file storage snapshot [ENABLE]."""
+
+import click
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import exceptions
+
+
+@click.command()
+@click.argument('volume_id')
+@click.option('--schedule-type',
+              help='Snapshot schedule [HOURLY|DAILY|WEEKLY]',
+              required=True)
+@click.option('--retention-count',
+              help='Number of snapshots to retain',
+              required=True)
+@click.option('--minute',
+              help='Minute of the day when snapshots should be taken',
+              default=0)
+@click.option('--hour',
+              help='Hour of the day when snapshots should be taken',
+              default=0)
+@click.option('--day-of-week',
+              help='Day of the week when snapshots should be taken',
+              default='SUNDAY')
+@environment.pass_env
+def cli(env, volume_id, schedule_type, retention_count,
+        minute, hour, day_of_week):
+    """Enables snapshots for a given volume on the specified schedule"""
+    file_manager = SoftLayer.FileStorageManager(env.client)
+
+    valid_days = {'SUNDAY', 'MONDAY', 'TUESDAY', 'WEDNESDAY', 'THURSDAY',
+                  'FRIDAY', 'SATURDAY'}
+
+    if (schedule_type != 'HOURLY' and schedule_type != 'DAILY'
```

Comment:
Let's move this to a list with 'if in' as we did with day of the week to keep it consistent